Podcast/AppDelegate.swift

```diff
@@ -30,7 +30,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         discoverViewController = DiscoverViewController()
         feedViewController = FeedViewController()
-        profileViewController = ProfileViewController()
+//        profileViewController = ProfileViewController()
```

Comment:
delete this comment, also maybe we should change profileviewcontroller to externalprofileviewcontroller for naming consistency